x-pack/plugins/security_solution/scripts/endpoint/common/fleet_services.ts

```diff
@@ -246,8 +256,30 @@ export const fetchAgentPolicyList = async (
       },
       query: options,
     })
-    .catch(catchAxiosErrorFormatAndThrow)
-    .then((response) => response.data);
+    .then((response) => response.data)
+    .catch(catchAxiosErrorFormatAndThrow);
+};
+
+/**
+ * Retrieves a list of Fleet Integration policies
+ * @param kbnClient
+ * @param options
+ */
+export const fetchIntegrationPolicyList = async (
+  kbnClient: KbnClient,
+  options: GetPackagePoliciesRequest['query'] = {}
+): Promise<GetPackagePoliciesResponse> => {
+  return kbnClient
+    .request<GetPackagePoliciesResponse>({
+      method: 'GET',
+      path: PACKAGE_POLICY_API_ROUTES.LIST_PATTERN,
+      headers: {
+        'elastic-api-version': '2023-10-31',
```

Comment:
Should we use the `API_VERSIONS.public.v1` from fleet plugin?

Reply:
See my prior comment to @tomsonpl 's (same comment). I never really understood why we are using `const`'s for API version